### ISSUE TITLE
Response object after login

### ISSUE THREAD

**Issue by jimilucio** [ngrome/codebattle-nodejs#1]
@LordKenzo As you can see, login operation return an object with password information. To improve the security context we've to avoid to return password as response.

**Comment by LordKenzo** [comment]
Yes, this is a **security issue**. Password should never goes within response token.

**Comment by LordKenzo** [comment]
We can leave for didactical purpose.
